packages/flutter/lib/src/painting/circle_border.dart

```diff
@@ -0,0 +1,98 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math' as math;
+
+import 'basic_types.dart';
+import 'borders.dart';
+import 'edge_insets.dart';
+
+/// A border that fits a circle within the available space.
+///
+/// Typically used with [ShapeDecoration] to draw a circle.
+///
+/// The [dimensions] assume that the border is being used in a square space.
+/// When applied to a rectangular space, the border paints in the center of the
+/// rectangle.
+///
+/// See also:
+///
+///  * [BorderSide], which is used to describe each side of the box.
+///  * [Border], which, when used with [BoxDecoration], can also
+///    describe a circle.
+class CircleBorder extends ShapeBorder {
+  /// Create a circle border.
+  ///
+  /// The argument must not be null.
```

Comment:
argument --> side argument

(so that if other arguments are added later this comment doesn't need to be updated)

Reply:
done